lib/glific/seeds/seeds_migration.ex

```diff
@@ -55,6 +55,7 @@ defmodule Glific.Seeds.SeedsMigration do
   defp do_migrate_data(:optin, organizations), do: optin_data(organizations)
   defp do_migrate_data(:opt_in_out, organizations), do: SeedsFlows.opt_in_out_flows(organizations)
   defp do_migrate_data(:simulator, organizations), do: add_simulators(organizations)
+  defp do_migrate_data(:template, organizations), do: SeedsFlows.add_template_flows(organizations)
```

Comment:
Shouldn't we add `migrate_data(:template, organization)` in `seeds/20200723172939_add_glific_data.exs`, so that it will work for new orgs?